script.js

```diff
@@ -0,0 +1,127 @@
+// HTML Elements
+const statusDiv = document.querySelector('.status');
+const resetDiv = document.querySelector('.reset');
+const cellDivs = document.querySelectorAll('.game-cell');
+
+// game constants
+const xSymbol = '×';
+const oSymbol = '○';
+
+// game variables
+let gameIsLive = true;
+let xIsNext = true;
+
+
+// functions
+const letterToSymbol = (letter) => letter === 'x' ? xSymbol : oSymbol;
+
+const handleWin = (letter) => {
+  gameIsLive = false;
+  if (letter === 'x') {
+    statusDiv.innerHTML = `${letterToSymbol(letter)} has won!`;
+  } else {
+    statusDiv.innerHTML = `<span>${letterToSymbol(letter)} has won!</span>`;
+  }
+};
+
+const checkGameStatus = () => {
+  const topLeft = cellDivs[0].classList[1];
```

Comment:
Checking the class makes sense, but if this thing got other classes added to it, then the class you're looking for would no longer be at index 1. I think this could be improved using `.classlist.contains('class-you-are-looking-for')`